Libplanet/Tx/TxFailure.cs

```diff
@@ -44,12 +47,18 @@ public sealed class TxFailure : TxExecution
         /// that the <see cref="Transaction{T}"/> is executed within.</param>
         /// <param name="txId">The executed <see cref="Transaction{T}"/>'s <see
         /// cref="Transaction{T}.Id"/>.</param>
+        /// <param name="actionsLogsList">The logs recorded while executing actions.</param>
         /// <param name="exception">The uncaught exception thrown by an action in the transaction.
         /// </param>
-        public TxFailure(BlockHash blockHash, TxId txId, Exception exception)
+        public TxFailure(
+            BlockHash blockHash,
+            TxId txId,
+            List<List<string>>? actionsLogsList,
+            Exception exception)
```

Comment:
As constructors of `TxFailure` and `TxSuccess` changed their signature, you need to record the changelogs for them.

Reply:
I added the below lines on changelogs

```markdown
 -  Added `actionsLogsList` parameter to `TxSuccess` constructor.
    [[#2474], [#2505]]
 -  Added `actionsLogsList` parameter to `TxFailure` constructor.
    [[#2474], [#2505]]
```